frontend/packages/react-app/src/components/pages/WithdrawAudiusPage.jsx

```diff
@@ -106,25 +117,28 @@ const WithdrawAudiusPage = () => {
   const addressSubmit = async (e) => {
```

Comment:
Following error occurred on submitting with address  `0xe2618c9ae9d8b30f5c36c3cabeed1e0175a64fbb`
> Unhandled Rejection (Error): call revert exception (method="distributedAmount(address)", errorSignature="Error(string)", errorArgs=["SafeMath: division by zero"], reason="SafeMath: division by zero", code=CALL_EXCEPTION, version=abi/5.0.7)

Reply:
This is the part related to `requestCheckingAddress`, so I'll do it in another PR

Reply:
OK!